web/src/components/zonemap.jsx

```diff
@@ -4,6 +4,7 @@ import ReactMapGL, { NavigationControl, Source, Layer } from 'react-map-gl';
 import { noop } from '../helpers/noop';
```

Comment:
_(Not related to this file, just adding it somewhere)_

The production/consumption toggle no longer works - is this on purpose? :)

Reply:
It works for me? 

It doesn't change the colors immediately, but neither does the current version